<?xml version="1.0" encoding="utf-8"?>
<ui version="4.0">
  <class>notepadClass</class>
  <widget class="QMainWindow" name="notepadClass">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>627</width>
        <height>400</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>notepad</string>
    </property>
    <widget class="QWidget" name="centralWidget">
      <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
          <rect>
            <x>10</x>
            <y>0</y>
            <width>601</width>
            <height>351</height>
          </rect>
        </property>
      </widget>
    </widget>
    <widget class="QMenuBar" name="menuBar">
      <property name="geometry">
        <rect>
          <x>0</x>
          <y>0</y>
          <width>627</width>
          <height>21</height>
        </rect>
      </property>
      <widget class="QMenu" name="menuFile">
        <property name="title">
          <string>File</string>
        </property>
        <addaction name="actionNew"/>
        <addaction name="actionOpen"/>
        <addaction name="actionSave"/>
        <addaction name="actionSave_As"/>
      </widget>
      <widget class="QMenu" name="menuEdit">
        <property name="title">
          <string>Edit</string>
        </property>
        <addaction name="actionCopy"/>
        <addaction name="actionCut"/>
        <addaction name="actionPaste"/>
        <addaction name="actionUndo"/>
        <addaction name="actionRedo"/>
      </widget>
      <addaction name="menuFile"/>
      <addaction name="menuEdit"/>
    </widget>
    <widget class="QToolBar" name="mainToolBar">
      <property name="iconSize">
        <size>
          <width>30</width>
          <height>30</height>
        </size>
      </property>
      <property name="toolButtonStyle">
        <enum>Qt::ToolButtonIconOnly</enum>
      </property>
      <attribute name="toolBarArea">
        <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
        <bool>false</bool>
      </attribute>
      <addaction name="actionNew"/>
      <addaction name="actionOpen"/>
      <addaction name="actionSave"/>
      <addaction name="actionSave_As"/>
      <addaction name="separator"/>
      <addaction name="actionCopy"/>
      <addaction name="actionCut"/>
      <addaction name="actionPaste"/>
      <addaction name="actionUndo"/>
      <addaction name="actionRedo"/>
    </widget>
    <action name="actionNew">
      <property name="icon">
        <iconset>
          <normaloff>Resources/new.png</normaloff>Resources/new.png
        </iconset>
      </property>
      <property name="text">
        <string>New</string>
      </property>
    </action>
    <action name="actionOpen">
      <property name="icon">
        <iconset>
          <normaloff>Resources/open.png</normaloff>Resources/open.png
        </iconset>
      </property>
      <property name="text">
        <string>Open</string>
      </property>
    </action>
    <action name="actionSave">
      <property name="icon">
        <iconset>
          <normaloff>Resources/save.png</normaloff>Resources/save.png
        </iconset>
      </property>
      <property name="text">
        <string>Save</string>
      </property>
    </action>
    <action name="actionSave_As">
      <property name="icon">
        <iconset>
          <normaloff>Resources/saveas.png</normaloff>Resources/saveas.png
        </iconset>
      </property>
      <property name="text">
        <string>Save As</string>
      </property>
    </action>
    <action name="actionCopy">
      <property name="icon">
        <iconset>
          <normaloff>Resources/copy.png</normaloff>Resources/copy.png
        </iconset>
      </property>
      <property name="text">
        <string>Copy</string>
      </property>
    </action>
    <action name="actionCut">
      <property name="icon">
        <iconset>
          <normaloff>Resources/cut.png</normaloff>Resources/cut.png
        </iconset>
      </property>
      <property name="text">
        <string>Cut</string>
      </property>
    </action>
    <action name="actionPaste">
      <property name="icon">
        <iconset>
          <normaloff>Resources/paste.png</normaloff>Resources/paste.png
        </iconset>
      </property>
      <property name="text">
        <string>Paste</string>
      </property>
    </action>
    <action name="actionUndo">
      <property name="icon">
        <iconset>
          <normaloff>Resources/undo.png</normaloff>Resources/undo.png
        </iconset>
      </property>
      <property name="text">
        <string>Undo</string>
      </property>
    </action>
    <action name="actionRedo">
      <property name="icon">
        <iconset>
          <normaloff>Resources/redo.png</normaloff>Resources/redo.png
        </iconset>
      </property>
      <property name="text">
        <string>Redo</string>
      </property>
    </action>
  </widget>
  <layoutdefault spacing="6" margin="11"/>
  <resources>
    <include location="notepad.qrc"/>
  </resources>
  <connections/>
</ui>
